src/chigame/games/models.py

```diff
@@ -6,13 +6,95 @@
 
 class Game(models.Model):
     """
-    A game (Chess, Checkers, etc.)
+    A game like Chess, Checkers, etc.
     """
 
-    name = models.TextField()
+    # Basic information
+    name = models.CharField(max_length=255)
     description = models.TextField()
+    image = models.URLField(default="/static/images/no_picture_available.png")
+    year_published = models.PositiveIntegerField(null=True)
+
+    # Gameplay information
+    rules = models.TextField(null=True)
+
     min_players = models.PositiveIntegerField()
     max_players = models.PositiveIntegerField()
+    suggested_age = models.PositiveSmallIntegerField(
+        null=True
+    )  # Minimum recommendable age. For example, 8+ would be 8.
+
+    expected_playtime = models.PositiveIntegerField(null=True)  # In Minutes
+    min_playtime = models.PositiveIntegerField(null=True)
+    max_playtime = models.PositiveIntegerField(null=True)
+
+    complexity = models.PositiveSmallIntegerField(null=True)  # 1-5, 1 being the easiest
+    category = models.ManyToManyField("Category", related_name="games")
+    mechanics = models.ManyToManyField("Mechanic", related_name="games")
+
+    # BGG information
+    BGG_id = models.PositiveIntegerField(null=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Person(models.Model):
+    """
+    A person associated with a game, such as a designer or artist.
+    """
+
+    DESIGNER = 1
+    ARTIST = 2
+
+    ROLE_CHOICES = (
+        (DESIGNER, "Designer"),
+        (ARTIST, "Artist"),
+    )
+
+    name = models.CharField(max_length=255)
```

Comment:
Is there a particular reason you're setting this max length?

Reply:
I was swayed by this

How to use CharField ?
CharField is used for storing small sized strings in the database. One can store First Name, Last Name, Address Details, etc. CharField should be given an argument max_length for specifying the maximum length of string it is required to store. In production server, after the Django application is deployed, space is very limited. So it is always optimal to use max_length according to the requirement of the field. Let us create an instance of the CharField we created and check if it is working.

from here:

https://www.geeksforgeeks.org/charfield-django-models/#

Reply:
I talked with borja about this, he stated that in modern day string limits are not particularly necessary since fixed length storage optimizations are no longer that important.
For the purposes of chigame, we can leave the `CharField`'s without `max_length` option.